package.json

```diff
@@ -42,12 +44,24 @@
   "devDependencies": {
     "@babel/core": "^7.20.12",
     "@polkadot/dev": "^0.67.177",
+    "@storybook/addon-essentials": "^7.0.12",
+    "@storybook/addon-interactions": "^7.0.12",
+    "@storybook/addon-links": "^7.0.12",
+    "@storybook/addon-styling": "^1.0.8",
+    "@storybook/blocks": "^7.0.12",
```

Comment:
Not sure, but I think we don't need that if we're not using the SB docs functionality?